app/uk/gov/hmrc/cgtpropertydisposalsstubs/controllers/SubscriptionUpdateProfiles.scala

```diff
@@ -35,20 +34,7 @@ object SubscriptionUpdateProfiles {
   def updateSubscriptionDetails(id: String): Option[SubscriptionUpdateProfile] =
     updates.find(_.predicate(id))
 
-  val desUpdate = SubscriptionUpdateResponse(
```

Comment:
we don't need this test case here - if the cgt reference is not one of the error ones below we can just return an ok response with the details the request stuck into the response